extendify/extendable
# Add FusionSky style and Fixes

This PR:

- Adds [FusionSky style variation](https://github.com/extendify/company-product/issues/562): Introducing the FusionSky style variation, a versatile and contemporary design tailored for modern corporate websites. This visually engaging theme is ideal for industries such as finance, banking, consulting, and other professional services.
- Optimizes headers with center logo: Site title removed for a cleaner look.
- Enhanced WooCommerce input consistency and deprecated custom button style for WooCommerce buttons as version 7.0.1 now supports style variation's button style.
- Refines search block style: Aligns visual design with overall UI.
- Expanded button styles and interactivity in all style variations: Added hover, focus, and active effects to buttons, enhancing their visual appeal and user interaction.
 
**How to test it best:**

- Activate FusionSky style: Enable the FusionSky style variation and examine your website for a seamless design and consistent appearance.
- Search block: Insert the search block onto a page and evaluate its visual integration and alignment with the overall style.
- Evaluate button interactivity: Previously, buttons lacked noticeable hover effects.  Insert the button block onto a page Interact with buttons to check the newly added hover, focus, and active effects, ensuring an improved visual appeal and user experience.
